src/test/java/examples/websocket/TradeStreamWithAllCallbacks.java

```diff
@@ -0,0 +1,44 @@
+package examples.websocket;
+
+import com.binance.connector.client.impl.WebsocketClientImpl;
+import com.binance.connector.client.utils.WebSocketCallback;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TradeStreamWithAllCallbacks {
+    private static final Logger logger = LoggerFactory.getLogger(TradeStreamWithAllCallbacks.class);
+    private static volatile boolean tradeStreamIsUp = false;
```

Comment:
Naming convention for boolean in Java should be prefixed with 'is'. Suggested variable name: isTradeStreamUp